.changeset/cool-singers-draw.md

```diff
@@ -0,0 +1,7 @@
+---
+'@hashicorp/react-toggle': major
+---
+
+- 💥✨ BREAKING CHANGE: Converts to CSS modules.
+  - Consumers will need to remove any `@hashicorp/react-checkbox-input/style.css` imports.
+  - Note: does _not_ support a `className` prop, as this component did not use any overrides previously.
```

Comment:
This component was one exception where `className` prop support did not seem necessary. So, have left it out for now.